docker-images/kafka/Dockerfile

```diff
@@ -49,6 +56,10 @@ COPY ./tmp/tracing-agent.jar ${KAFKA_HOME}/libs/
 #####
 COPY kafka-thirdparty-libs/${THIRD_PARTY_LIBS}/target/dependency/ ${KAFKA_HOME}/libs/
 
+RUN  mkdir -p ${CRUISE_CONTROL_HOME}/libs/
+COPY kafka-thirdparty-libs/cc/target/dependency/ ${CRUISE_CONTROL_HOME}/libs/
+COPY kafka-thirdparty-libs/cc/target/dependency/cruise-control-metrics-reporter-*.jar ${KAFKA_HOME}/libs/
```

Comment:
Why is this not just added to the third-party-libs for given version? You might anyway need to change it later once you need different versions for different broker.

Reply:
I'll take another look at this